docs/src/add-services/mysql/_index.md

```diff
@@ -346,6 +346,27 @@ ALTER TABLE table_name CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_
 
 For further details, see the [MariaDB documentation](https://mariadb.com/kb/en/character-set-and-collation-overview/).
 
+## Password generation
+ 
+When you create an endpoint to allow your app and database to communicate,
+an empty password is generated by default.
+If you add multiple endpoints,
+having empty passwords can cause technical issues with your app.
+ 
+To solve this,
+define custom endpoints in your [service configuration](#configuration-options),
+similarly to when you create [multiple databases](#multiple-databases).
+For each custom endpoint you create, 
+you get an automatically generated password.
+After your custom endpoints are exposed as relationships in your [app configuration](../../create-apps/_index.md),
+you can retrieve the password for each endpoint
+through the `PLATFORM_RELATIONSHIPS` [environment variable](../../development/variables/use-variables.md#use-platformsh-provided-variables).
```

Comment:
I left the first link unchanged because: 
- Before users can access the environment variable, imho I believe it's best if they at least have a rough idea of what it is.
- I added another sentence to explain how they can access the variable with links to both the shell method and the Config Reader method. I believe both sentences complement each other, what do you think?

Reply:
Yeah that definitely makes sense! Context first and then how to action that knowledge.